src/controller/handler/connections.ts

```diff
@@ -111,3 +111,17 @@ export class Connections extends BaseAgentResponseHandler {
         return;
     }
 }
+
+interface Body {
```

Comment:
if we intend to have this interface, we need to put in its own file and follow our pattern for declaring interfaces

Reply:
@matt-raffel-kiva TBH this repo probably needs a good deal of refactoring. I'm with you on pulling this out though. In general, I would like to see **a lot** more use of types and type validations in this repo.

Reply:
Agreed.  The challenge comes from Acapy which is calling us with the data structures and it is not easily identifiably consistent as the structures vary depending on factors like where the message originates.  I will write up an architectural issue for us to work on.